rpcserver.go

```diff
@@ -5483,20 +5471,29 @@ func (s *rpcServer) Start() {
 		})
 		s.WebsocketHandler(ws, r.RemoteAddr, authenticated, isAdmin)
 	})
+	return httpServer
+}
 
+// Run  is used by server.go to start the rpc listener.
```

Comment:
Update the comment to reflect the fact that run blocks until the context is canceled.